src/client/app/index.jsx

```diff
@@ -25,28 +23,57 @@ class App extends React.Component {
     let context = this;
     $.ajax({
       type: 'GET',
-      url: '/listings/21/reviews',
+      url: '/listings/134/reviews',
       success: function(data) {
-        console.log('success', data);
-        context.setState({'reviews': data});
         context.getDistribution(data);
         context.setState({reviews: data});
         context.setState({copied: data});
-        let words = context.getWords(data);
-        context.setState({topWords: words});
+        context.getWords(data);
       },
       error: function(err) {
         console.log('error');
       }
     })
   }
-  
+
+  generateGraphData(count, percents) {
+    let rating = {
+      5: 'Excellent',
+      4: 'Very good',
+      3: 'Average',
+      2: 'Poor',
+      1: 'Terrible'
+    }
+    let info = [];
+    for (let key in count) {
+      info.unshift({rating: rating[key], count: count[key], percentage: percents[key], rank: key});
+    }
```

Comment:
This looks risky to me - but I remember James Powers saying that es6 handles for / in on arrays differently now. But on a first read it looks like something that isn't guaranteed to produce the same order every time. If you can prove me wrong please do, I'd appreciate it !!